src/components/templates/Search/utils.ts

```diff
@@ -4,23 +4,45 @@ import {
 } from '@oceanprotocol/lib/dist/node/metadatacache/MetadataCache'
 import { MetadataCache, Logger } from '@oceanprotocol/lib'
 
+const SortTermOptions = {
+  Liquidity: 'liquidity',
+  Price: 'price',
+  Created: 'created',
+} as const
+type SortTermOptions = typeof SortTermOptions[keyof typeof SortTermOptions]
+
+const SortValueOptions = {
+  Ascending: 'asc',
+  Descending: 'desc',
+} as const
+type SortValueOptions = typeof SortValueOptions[keyof typeof SortValueOptions]
+
 export function getSearchQuery(
   text?: string,
   owner?: string,
   tags?: string,
   categories?: string,
   page?: string,
-  offset?: string
+  offset?: string,
+  sort?: string,
+  sortOrder?: string,
+  priceType?: string
 ): SearchQuery {
-  const searchTerm = owner
+  
+  const sortTerm = ( sort == SortTermOptions.Liquidity ) ? "price.ocean" : (( sort == SortTermOptions.Price ) ? "price.value" : SortTermOptions.Created)
+  const sortValue = ( sortOrder == SortValueOptions.Ascending ) ? 1 : -1
+
+  let searchTerm = owner
     ? `(publicKey.owner:${owner})`
     : tags
     ? // eslint-disable-next-line no-useless-escape
       `(service.attributes.additionalInformation.tags:\"${tags}\")`
     : categories
     ? // eslint-disable-next-line no-useless-escape
       `(service.attributes.additionalInformation.categories:\"${categories}\")`
-    : text || ''
+    : text || '';
+
+  searchTerm = (priceType) ? searchTerm+' AND price.type:'+priceType : searchTerm
```

Comment:
we usually use template literals to combine strings, so:

```suggestion
  searchTerm = priceType ? `${searchTerm} AND price.type:${priceType}` : searchTerm
```

Reply:
yeah sure, I'll modify using your suggestion.